lib/html_parser.dart

```diff
@@ -76,15 +80,21 @@ class HtmlParser extends StatelessWidget {
     // using textScaleFactor = 1.0 (which is the default). This ensures the correct
     // scaling is used, but relies on https://github.com/flutter/flutter/pull/59711
     // to wrap everything when larger accessibility fonts are used.
-    return StyledText(
-      textSpan: parsedTree,
-      style: cleanedTree.style,
-      textScaleFactor: MediaQuery.of(context).textScaleFactor,
-      renderContext: RenderContext(
-        buildContext: context,
-        parser: this,
-        style: Style.fromTextStyle(Theme.of(context).textTheme.bodyText2),
-      ),
+    return Scrollable(
```

Comment:
I wonder what the impact of this is. 'Suddenly' we are now always a scrollable container, even in shrinkWrap mode. I don't quite know yet if this really _is_ a problem, but it surely impacts how our `Html` widget is to be used right?

Reply:
I don't think shrinkWrap mode affects it, the scrollable is the same API that a Column or ListView uses, so it can still be used in other Flex widgets with no problems. As the docs say:

> Scrollable implements the interaction model for a scrollable widget, including gesture recognition, but does not have an opinion about how the viewport, which actually displays the children, is constructed.

Therefore I don't think its a huge issue.